recipes-connectivity/prplmesh/prplmesh.bb

```
@@ -8,27 +8,108 @@ inherit cmake pythonnative
 S = "${WORKDIR}/git"
 OECMAKE_SOURCEPATH = "${S}/"
 
+SUMMARY = "prplMesh"
 SECTION = "net"
 LICENSE = "BSD-2-Clause-Patent.txt"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=7b45146d47e73bcbac1068e5cfc2a9fb"
-DEPENDS = "python-native python-pyyaml-native json-c openssl zeromq"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=0518d409dae93098cca8dfa932f3ab1b"
+
+DEPENDS = "python-native python-pyyaml-native json-c openssl"
 RDEPENDS_${PN} = "iproute2 busybox readline"
-SUMMARY = "prplMesh"
 
-SRCREV = "911392026280d9961823a9359e6cc4f08c9ee0a2"
-SRC_URI = "git://github.com/prplfoundation/prplMesh.git;protocol=http;branch=master"
+SRCREV = "8a331d9fa1631782cf06603ee612ce6133ee0c1a"
+SRC_URI = "git://github.com/prplfoundation/prplMesh.git;protocol=http;branch=master \
+           file://001-add_find_nl_for_nl80211_bwl_type.patch \
+           file://002-allow_redefine_beerocks_vars_from_cmdline.patch \
+           file://003_prplmesh_util_stop_use_hard_coded_wlan_iface_names.patch \
+           file://004-get_hostapd_ctrl_iface_pathes_from_conf.patch"
+
+
+PACKAGECONFIG ??= "${BWL_TYPE} ${MSGLIB}"
 
-BUILD_TYPE ?= "Release"
 TARGET_PLATFORM ?= "linux"
-BWL_TYPE ?= "DUMMY"
+MSGLIB ?= "mq-zmq"
+BWL_TYPE ?= "bwl-nl80211"
 INSTALL_PREFIX ?= "/opt/prplmesh"
+#
+# QEMU MACHINE specific
+#
+BWL_TYPE_qemuarm ?= "bwl-dummy"
+BWL_TYPE_qemuarm64 ?= "bwl-dummy"
+BWL_TYPE_qemumips ?= "bwl-dummy"
+BWL_TYPE_qemumips64 ?= "bwl-dummy"
+BWL_TYPE_qemuppc ?= "bwl-dummy"
+BWL_TYPE_qemux86 ?= "bwl-dummy"
+BWL_TYPE_qemux86-64 ?= "bwl-dummy"
+#
+# RPi machine specific
+#
+# "wlan2" is not activate on RPi. So it's needed to plug-in Wi-Fi
+# USB dongle to have two wlan ifaces: wlan0 and wlan1
+BEEROCKS_WLAN1_IFACE_raspberrypi-rdk-broadband = "wlan0"
+BEEROCKS_WLAN2_IFACE_raspberrypi-rdk-broadband = "wlan1"
+BEEROCKS_HOSTAP_WLAN1_CTRL_IFACE_raspberrypi-rdk-broadband="/var/run/hostapd0/wlan0"
+BEEROCKS_HOSTAP_WLAN2_CTRL_IFACE_raspberrypi-rdk-broadband="/var/run/hostapd4/wlan1"
```

```suggestion
BEEROCKS_HOSTAP_WLAN2_CTRL_IFACE_raspberrypi-rdk-broadband="/var/run/hostapd1/wlan1"
```
Why `hostapd4` in the path?

---

It's RPi specific